### PR TITLE
Site titleand logo#38

### PR DESCRIPTION
**Pull request by AndrewPolanyi**
Not sure if I made a mistake pushing the package-lock.json file. But the pull from master this morning was not working for me so i needed to do a fresh npm install for dependencies.

changed title and favicon, I found that the stencil version of the logo looked worse as a favicon but I added the logo incase we want to switch.

### CODE REVIEW

**Comment by AndrewPolanyi** [comment]
image
or
image

File: src/components/Nav.tsx [hunk 1]
```diff
@@ -96,11 +97,31 @@ export const NavHeader = () => {
...
+            padding="5px" 
+          >
+          </Image>
+        </Box>
+
+        {/* added some padding between logo and home*/}
+        <Box
+          width="15px"
+        >
+        </Box>
```
Comment (jayden-chan) [inline src/components/Nav.tsx]: You should just put a `mr="15px"` on the other box above instead

**Comment by jayden-chan** [comment]
Looks good, maybe some vertical margin on the logo would be nice so it's not touching the top and bottom of the navbar

File: src/components/Nav.tsx (hunk 1 above)
Comment (AndrewPolanyi) [reply src/components/Nav.tsx]: thanks!

**Comment by AndrewPolanyi** [comment]
Thanks @jayden-chan, looks better now 
image

**Review by jayden-chan — DISMISSED** [review]

File: src/components/Nav.tsx [hunk 2]
```diff
@@ -96,11 +97,28 @@ export const NavHeader = () => {
...
+          h="40px"
+          w="40px"
+          borderRadius="50%"
+        >
+          <Image 
+            src="/logo.png" 
+            alt="schedulator logo" 
+            height="40px"
+            width="40px"
+            minWidth="40px"
```
Comment (nicudy) [inline src/components/Nav.tsx]: A bit nit picky but I don't think you actually need to specify height and width here, the logo should take the size of the box automatically. minWidth can be kept if you want to ensure the logo stays the same size when the window changes, however.

Comment (nicudy) [reply src/components/Nav.tsx]: Looks good otherwise!

**Review by nicudy — DISMISSED** [review]

File: src/components/Nav.tsx (hunk 2 above)
Comment (AndrewPolanyi) [reply src/components/Nav.tsx]: sweet thanks!

**Review by Adamk8 — APPROVED** [review]
Good work!

**Review by jayden-chan — APPROVED** [review]

**Pull request merged**